### DISCUSSION TITLE
copy files local to a document library

### DISCUSSION THREAD

**Discussion by jasonsigurdur** [abraunegg/onedrive#1320] [Q&A] [answered]
would it be possible to copy local files to a document library?

**Comment by abraunegg** [comment, marked as answer]
@jasonsigurdur 
This is possible, refer to https://github.com/abraunegg/onedrive/blob/master/docs/SharePoint-Shared-Libraries.md

**Reply by jasonsigurdur** [reply]
The document shows Sharepoint sites as shared libraries. I am trying to sync a 'Document library' within the Sharepoint site.

**Reply by abraunegg** [reply]
@jasonsigurdur 
It is still the same process - please follow the document and configure your client to use the Document Library you wish to sync to.

**Reply by jasonsigurdur** [reply]
Ok thank you I will give it a try.
